### ISSUE TITLE
UnityTiles Names

### ISSUE THREAD

**Issue by Flameshot** [mapbox/mapbox-unity-sdk#1249]
_Note: If this is a bug or support ticket, please provide the following information:_
* Unity version:  2018.2.6f1
* Mapbox SDK version: v2.0.0
* The platform you're building to: Widnows
* A description of what you're trying to do: 
Hi guys. I made a custom feature to draw polygons on map using Triangle.NET and after I triangulate them, my system is made in a way that the polygons will become a child of the tile where they are drawn. My problem now is that I want to find a way to have an unique name for UnityTiles, because I saw the name for the tiles is reused and when I start to move on the map (having Camera Bounds extent option) the drawn polygon will appear on another part of map, because the parent has the same name as the first tile where I drawn the polygon. I hope I made myself understood. Any ideas?

**Comment by brnkhy** [comment]
@Flameshot I'm not sure what you mean. Tiles are already named uniquely by their TileId;
![unity_2019-01-21_19-06-28](https://user-images.githubusercontent.com/1465390/51485670-b71b1b00-1daf-11e9-8939-f66bdc53e5c2.png)

Tiles are recycled though so if you out something under it, it might reappear when it's reused. If that's what you mean, you might want to consider:
- not putting them under the unity tile object, does it have to be there?
- destroy it when tile is unloaded/recycled
- move it out of the hierarchy when tile is unloaded/recycled

We dont have helper events for tile unload if I remember correct but you can check `TriggerTileRedrawForExtent` method under AbstractMap and modify it as tile unloading procedure starts there.

**Comment by Flameshot** [comment]
@brnkhy Thanks for answearing.

I saw that their names are reused/recycled. When I draw a polygon, it becomes a child of that tile where it was drawn. Now, I'm looking for a way to disable the reusing of the same tile name, because when it will be reused, if that tile has a drawn polygon as a child, the polygon will be positioned where the recycled tile name will be used.

I made the draw system this way:
1. Draw polygon with mouse clicks.
2. Triangulate (transform to a polygon having input vertices)
3. Becomes child of the tile where it was drawn to know if we go back to this tile, the polygon will be drawn again in the same place.

Maybe now you understand more why I made things this way.
